canopen/node/remote.py

```diff
@@ -89,27 +89,46 @@ def restore(self, subindex=1):
         """
         self.sdo.download(0x1011, subindex, b"load")
 
+    def __load_configuration_helper(self, index, subindex, name, value):
+        """Helper function to send SDOs to the remote node
+        :param index: Object index
+        :param subindex: Object sub-index (if it does not exist e should be None)
+        :param name: Object name
+        :param value: Value to set in the object
+        """
+        try:
+            if subindex is not None:
+                logger.info(str('SDO [{index}][{subindex}]: {name}: {value}'.format(
+                    index=format(index, '#04x'),
+                    subindex=subindex,
+                    name=name,
+                    value=format(value, '#04x'))))
```

Comment:
You could include the format of the hex in the first string: `logger.info('SDO [{index:#06x}][{subindex}]: {name}: {value:#06x}' ...`.

Reply:
Ok, done!